incubator/redis-cache/values.yaml

```diff
@@ -0,0 +1,22 @@
+# Default values for redis-cache.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicaCount: 3
+image:
+  repository: nginx
+  tag: stable
+  pullPolicy: IfNotPresent
+service:
+  name: rd-port
+  Port: 6379
+  type: ClusterIP
+podDisruptionBudget:
+  minAvailable: 2
+resources:
```

Comment:
It is best practice not to set defaults for resources but to make them indeed configurable.